### PR TITLE
Various updates for NAT64

### PR DESCRIPTION
**Pull request by laurentderu**
This PR contains various small updated for the NAT64 modules :

* Support for runtime configuration of the NAT64 prefix used, but default it's still the ::FFFF:/96 prefix
* Removal of the servername in the outgoing DHCP request
* Replace printf() with PRINTF() macros
* Update of the nameserver using the information received in the DHCP reply (Note that the update is only done if no server is already configured or if it was already configured by NAT64)

### CODE REVIEW

File: core/net/ip64/ip64-ipv4-dhcp.c [hunk 1]
```diff
@@ -58,10 +61,10 @@ PROCESS_THREAD(ip64_ipv4_dhcp_process, ev, data)
 
   ip64_dhcpc_init(&ip64_eth_addr, sizeof(ip64_eth_addr));
 
-  printf("Inited\n");
+  PRINTF("Inited\n");
 
   ip64_dhcpc_request();
-  printf("Requested\n");
+  PRINTF("Requested\n");
```
Comment (alignan) [inline core/net/ip64/ip64-ipv4-dhcp.c]: Maybe we should change the prints to something more meaningful i.e `IP64: Initited`, etc

File: core/net/ip64/ip64-ipv4-dhcp.c [hunk 2]
```diff
@@ -78,15 +81,22 @@ void
...
 	 s->ipaddr.u8[0], s->ipaddr.u8[1],
 	 s->ipaddr.u8[2], s->ipaddr.u8[3]);
 
   ip64_set_hostaddr((uip_ip4addr_t *)&s->ipaddr);
   ip64_set_netmask((uip_ip4addr_t *)&s->netmask);
   ip64_set_draddr((uip_ip4addr_t *)&s->default_router);
-  ip64_addr_4to6((uip_ip4addr_t *)&s->dnsaddr, &ip6dnsaddr);
-  //  mdns_conf(&ip6dnsaddr);
+  if(!uip_ip4addr_cmp((uip_ip4addr_t *)&s->dnsaddr, &uip_all_zeroes_addr)) {
+    //Note: Currently we assume only one DNS server
```
Comment (alignan) [inline core/net/ip64/ip64-ipv4-dhcp.c]: Comments should be in `/* */`, also change below

**Comment by alignan** [comment]
Looks good!

File: core/net/ip64/ip64-ipv4-dhcp.c (hunk 1 above)
Comment (laurentderu) [inline core/net/ip64/ip64-ipv4-dhcp.c]: done

File: core/net/ip64/ip64-ipv4-dhcp.c (hunk 2 above)
Comment (laurentderu) [inline core/net/ip64/ip64-ipv4-dhcp.c]: done too

**Comment by adamdunkels** [comment]
:+1:

**Pull request merged**